frontend/src/metabase/lib/compat/check-version.ts

```diff
@@ -0,0 +1,4 @@
+export const isReactVersionLessThanOrEqualTo17 = (version: string) => {
```

Comment:
nit: couldn't this be a more generic "getMajorVersion" that returns a number, and then we can `major <= 17`?

Reply:
yes